corehq/apps/translations/tests/test_bulk_app_translation.py

```diff
@@ -38,6 +39,7 @@
 
 
 class BulkAppTranslationTestBase(SimpleTestCase, TestXmlMixin):
+    root = os.path.dirname(__file__)
```

Comment:
what does this change do?

Reply:
It changes the base directory for the `get_path` call in `do_upload` that locates the xlsx files. Ultimately it's used in test_utils.py: https://github.com/dimagi/commcare-hq/blob/326ff8bd5d422d8520a37e8b101da247e83d8731/corehq/util/test_utils.py#L87-L95